package.json

```diff
@@ -22,11 +22,17 @@
     "isomorphic"
   ],
   "author": "Reza Akhavan <jedireza@gmail.com> (http://reza.akhavan.me/)",
+  "contributors": [
+    "Tane Piper <piper.tane@gmail.com> (https://github.com/tanepiper)"
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jedireza/hapi-react-views/issues"
   },
   "homepage": "https://github.com/jedireza/hapi-react-views",
+  "engines": {
+    "node": ">=4.0.0"
+  },
```

Comment:
I wonder if we should include the `engines` property. Other dependencies (like `hapi`) already have this and are assumed to be around.